src/python/pants/core_tasks/reporting_server_run.py

```diff
@@ -1,72 +0,0 @@
-# Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
-# Licensed under the Apache License, Version 2.0 (see LICENSE).
-
-import logging
-
-from pants.base.exceptions import TaskError
-from pants.reporting.reporting_server import ReportingServerManager
```

Comment:
`ReportingServerManager` is only accessible via these tasks I think. cc @benjyw

Reply:
Agreed. I started to remove the whole file but stopped because some of it was used in `reporting.py` and it looked too intricate to remove in this PR.